tests/clone_test.go

```diff
@@ -692,6 +694,25 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 					expectEqualAnnotations(targetVMCloneFromClone, sourceVM)
 				})
 
+				It("VM with a hotplug disk", func() {
+					sourceVM = createVMWithStorageClass("rook-ceph-block-wffc", true)
+					sourceVM = StartVMAndExpectRunning(virtClient, sourceVM)
+
+					By("Add persistent hotplug disk")
```

Comment:
Not sure hotplug contributes/adds to the issue(s) here.  I see at least two things.

First is that the restored DataVolume is going  into `WaitForFirstConsumer` state and it should go into `PendingPopulation` instead.  This is a CDI bug.  This is not a "terminal" bug though.  Everything should still work but it is slightly more inefficient because kubevirt is using the "doppelganger" pod machinery.

The real issue appears to be that the vm clone controller is deleting the vmrestore before new PVC bound.  The PVC refers to a VolumeSnapshot that is owned by the VirtualMachineRestore.  So if the VirtualMachineRestore is deleted, the PVC will never be bound.

Suggested fix is for the clone controller to keep around VMRestores until all VM PVCs are bound.